email-service/src/main/java/com/vgur/spring/email/services/EmailSending.java

```diff
@@ -10,30 +12,59 @@
 import org.springframework.amqp.rabbit.core.RabbitTemplate;
 import org.springframework.amqp.rabbit.listener.SimpleMessageListenerContainer;
 import org.springframework.amqp.rabbit.listener.adapter.MessageListenerAdapter;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.mail.SimpleMailMessage;
 import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
+
 @Service
 @RequiredArgsConstructor
 @Slf4j
+@EnableConfigurationProperties(
+        EmailProperties.class
+)
 public class EmailSending {
     private final JavaMailSender mailSender;
+    private final EmailProperties emailProperties;
+    private ArrayList<OrderDto> ordersToSend;
+
+    @PostConstruct
+    public void init(){
+        ordersToSend = new ArrayList<>();
+    }
+
     @Bean
```

Comment:
why do u have @bean in that service ? maybe it's worth using configurations?

Reply:
yes I'll put it in configuration class